include/cuco/detail/static_set/static_set.inl

```diff
@@ -191,7 +191,56 @@ template <class Key,
           class Storage>
 template <typename InputIt, typename OutputIt>
 void static_set<Key, Extent, Scope, KeyEqual, ProbingScheme, Allocator, Storage>::contains_async(
-  InputIt first, InputIt last, OutputIt output_begin, cuda_stream_ref stream) const
+  InputIt first, InputIt last, OutputIt output_begin, cuda_stream_ref stream) const noexcept
+{
+  auto const num_keys = cuco::detail::distance(first, last);
+  if (num_keys == 0) { return; }
+
+  auto const grid_size =
+    (cg_size * num_keys + detail::CUCO_DEFAULT_STRIDE * detail::CUCO_DEFAULT_BLOCK_SIZE - 1) /
+    (detail::CUCO_DEFAULT_STRIDE * detail::CUCO_DEFAULT_BLOCK_SIZE);
+
+  auto const always_true = thrust::constant_iterator<bool>{true};
+  detail::contains_if_n<cg_size, detail::CUCO_DEFAULT_BLOCK_SIZE>
+    <<<grid_size, detail::CUCO_DEFAULT_BLOCK_SIZE, 0, stream>>>(
+      first, num_keys, always_true, thrust::identity{}, output_begin, ref(op::contains));
+}
+
+template <class Key,
+          class Extent,
+          cuda::thread_scope Scope,
+          class KeyEqual,
+          class ProbingScheme,
+          class Allocator,
+          class Storage>
+template <typename InputIt, typename StencilIt, typename Predicate, typename OutputIt>
+void static_set<Key, Extent, Scope, KeyEqual, ProbingScheme, Allocator, Storage>::contains_if(
+  InputIt first,
+  InputIt last,
+  StencilIt stencil,
+  Predicate pred,
+  OutputIt output_begin,
+  cuda_stream_ref stream) const
+{
+  contains_async(first, last, stencil, pred, output_begin, stream);
```

Comment:
Shouldn't this be `contains_if_async`?

Reply:
dang... the merge conflict got me

Reply:
Since you're going to make that last change, I'll approve the PR. Thanks!